src/models/ruleSection.js

```diff
@@ -0,0 +1,14 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var RuleSectionSchema = new Schema({
+  _id: {
+    type: String,
+    select: false
+  },
+  index: String,
+  name: String,
+  url: String
+});
+
+module.exports = mongoose.model('RuleSection', RuleSectionSchema, 'rules-sections');
```

Comment:
One last nit pick, and this will probably require a lot of find/replace. But what do you think about `rule-sections` instead of `rules-sections`?

Reply:
I actually agree, I actually struggled with naming it which is why its not consistent

Reply:
I think it makes the most sense for everything to be `rule-sections`, including the endpoint.

Reply:
so I just realized that the urls in the database are `/api/rules-sections` which is why `rules-sections` is everywhere in the api now

Reply:
here is the pr for the db if we want to go in that direction https://github.com/bagelbits/5e-database/pull/303

Reply:
Thanks!

Reply:
You should be good now.